apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -4,44 +4,40 @@ title: 'Compute Add-ons'
 description: "Learn about your project's instance and additional add-ons."
 ---
 
-Every project on the Supabase Platform comes with its own dedicated Postgres instance running inside a virtual machine (VM). The following table describes the base instance with additional compute add-ons available if you need extra performance when scaling up Supabase.
+Every project on the Supabase Platform comes with its own dedicated Postgres instance running inside a virtual machine (VM).
+
+The following table describes the base instance with additional compute add-ons available if you need extra performance when scaling up Supabase.
+
+| Compute Size    | Hourly Price USD | Monthly Price USD | CPU                     | Memory       | Database Max Connections[^1] | Connection Pooler Max Clients | Max DB Size (Recommended)[^2] |
+| --------------- | ---------------- | ----------------- | ----------------------- | ------------ | ---------------------------- | ----------------------------- | ----------------------------- |
+| Nano (free)[^3] | $0               | $0                | Shared                  | Up to 0.5 GB | 60                           | 200                           | 500MB                         |
```

Comment:
An important thing to add here is that on paid plans we incur $10 compute costs (same as micro) as minimum compute - this is missing entirely right now

Reply:
It might be better to remove Nano instance in this table entirely, as it's not a compute add-on - then mention the free micro upgrade at the bottom

Reply:
>It might be better to remove Nano instance in this table entirely

we can't do that b/c this table is the only place anywhere that mentions the Nano instance and its resources. i'll reframe the table so it's not just compute add ons.